playbooks/roles/airship-deploy-ucp/defaults/main.yml

```diff
@@ -2,3 +2,5 @@
 airship_pegleg_image: "{{ suse_airship_registry_location }}/airshipit/pegleg:{{ suse_airship_components_image_tag['pegleg'] }}"
 run_tests: true
 test_timeout: 2700
+neutron_external_interface: public
+neutron_tunnel_device: docker0
```

Comment:
Does it make sense to have `docker0` as a default value? I was under the impression that no value is better ...

Reply:
Yeah, I think it's better to leave it empty (or set to `null`) than to put `docker0` here. `docker0` would only work in single node deployments. But in that case neutron doesn't even need to create any tunnels.

Reply:
done